mz_strm_zstd.c

```diff
@@ -160,7 +160,7 @@ int32_t mz_stream_zstd_read(void *stream, void *buf, int32_t size) {
         zstd->total_in += in_bytes;
         zstd->total_out += out_bytes;
 
-    } while (zstd->in.pos < zstd->in.size && zstd->out.pos < zstd->out.size);
```

Comment:
Why this change?

Reply:
This is a previous modification and is currently consistent with upstream.